src/main.rs

```diff
@@ -638,14 +645,18 @@ fn process_yaml_test(filename: &str) {
         serde_yaml::from_slice::<TestFile>(&content[..]).expect("expected valid yaml");
     debug!("{:#?}", test_file);
 
-    let beacon_state: BeaconState = test_file.beacon_state.into();
-    let pre_state: ShardState = test_file.shard_pre_state.into();
-    let post_state: ShardState = test_file.shard_post_state.into();
+    let beacon_state: BeaconState = test_file.beacon_state.try_into().expect("");
```

Comment:
Can you add some error msg in the expects here and line 657?

Reply:
Will fix these in a subsequent PR.

Reply:
Actually fixed here.